graylog2-web-interface/packages/eslint-config-graylog/index.js

```diff
@@ -42,7 +42,10 @@ module.exports = {
       files: ['*.js', '*.jsx'],
     },
     {
-      files: ['*.test.js', '*.test.jsx', '*.test.ts', '*.test.tsx'],
+      files: [
+        '*.test.js', '*.test.jsx', '*.test.ts', '*.test.tsx',
+        '*.it.js', '*.it.jsx', '*.it.ts', '*.it.tsx',
```

Comment:
Probably only the `*.it.tsx` filename extension is relevant for integration tests, but I added the others for the sake of completeness.